Marketplace/README.md

```diff
@@ -0,0 +1,85 @@
+# Azure Stack Compute Administration
```

Comment:
Why is this ReadMe included in the PR? It looks to be outdated since it does not use version profiles to get the correct PS versions for Azure PS